doc/nrf/links.txt

```diff
@@ -415,6 +416,7 @@
 .. _`Migrating to the current hardware model`: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/migration/migration_hwmv2.html
 
 
+.. _`Migration notes for nRF Connect SDK v2.7.99-cs1 and the nRF54H20 DK`: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7_99-cs1.html
```

Comment:
The PR looks good to me, just leaving this comment as a reminder that after https://github.com/nrfconnect/sdk-nrf/pull/17015 gets merged, the new migration guide page should also be orphaned.